packages/site/src/main.css

```diff
@@ -158,3 +165,8 @@ iframe {
   width: 100%;
   height: 2vw;
 }
+
+.headerBlock > * {
```

Comment:
Kinda brittle but given our control over what this is, seems OK

Could throw a div/span with a classname around the targeted element for more granular targeting (already using `header` on the headerBlock itself)

Reply:
This site is pretty small so I'm guessing this would be okay, but since CSS targeting evaluates right to left, this seems like it would be slower than necessary?